src/Data/Textile/Process.elm

```diff
@@ -106,6 +109,11 @@ decode impactsDecoder =
         |> Pipe.required "alias" (Decode.maybe decodeAlias)
 
 
+getDisplayName : Process -> String
+getDisplayName process =
+    Maybe.withDefault process.name process.displayName
```

Comment:
So this is the same fn as the other for Food. Would it be possible to just have one generic, like:

```elm
getDisplayName : { displayName : Maybe String, name : String } -> String
getDisplayName { displayName, name } =
    Maybe.withDefault name displayName
```

Reply:
Actually I wouldn't have defined a function just for a withDefault.  I would rather remove it. The generic one looks more complicated for something clone to an identity: `getDisplayName {a, b} = withDefault a b`

Reply:
Yeah so please just use Maybe.withDefault instad of dedicated functions doing the same, thanks.

Reply:
Sorry I saw this after the merge. Will do another PR

Reply:
@ccomb no it's fine, leave it like that